octavia-cli/unit_tests/test_apply/test_commands.py

```diff
@@ -177,80 +204,52 @@ def test_create_resource(patch_click, mocker):
-    "force,diff,should_update_resource,expect_prompt,user_validate_diff,expect_update,expected_number_of_messages",
+    "force,diff,local_file_changed,expect_prompt,user_validation,expect_update",
     [
-        (True, True, True, False, False, True, 3),  # Force is on, we have a diff, prompt should not be displayed: we expect update.
-        (
-            True,
-            False,
-            True,
-            False,
-            False,
-            True,
-            3,
-        ),  # Force is on, no diff, should_update_resource == true, prompt should not be displayed, we expect update.
-        (
-            True,
-            False,
-            False,
-            False,
-            False,
-            False,
-            1,
-        ),  # Force is on, no diff, should_update_resource == false, prompt should not be displayed, we don't expect update. This scenario should not exists in current implementation as force always trigger update.
-        (
-            False,
-            True,
-            True,
-            True,
-            True,
-            True,
-            3,
-        ),  # Force is off, we have diff, prompt should be displayed, user validate diff: we expected update.
-        (
-            False,
-            False,
-            True,
-            False,
-            False,
-            True,
-            3,
-        ),  # Force is off, no diff, should_update_resource == true (in case of file change), prompt should not be displayed, we expect update.
-        (
-            False,
-            True,
-            True,
-            True,
-            False,
-            False,
-            1,
-        ),  # Force is off, we have a diff but the user does not validate it: we don't expect update.
-        (
-            False,
-            False,
-            False,
-            False,
-            False,
-            False,
-            1,
-        ),  # Force is off, we have a no diff, should_update_resource == false: we don't expect update.
+        pytest.param(True, True, True, False, False, True, id="Force, diff, local file change -> no prompt, no validation, expect update."),
+        pytest.param(
+            True, True, False, False, False, True, id="Force, diff, no local file change -> no prompt, no validation, expect update."
+        ),
+        pytest.param(
+            True, False, False, False, False, True, id="Force, no diff, no local file change -> no prompt, no validation, expect update."
+        ),
+        pytest.param(
+            True, False, True, False, False, True, id="Force, no diff, local file change -> no prompt, no validation, expect update."
+        ),
+        pytest.param(
+            False, True, True, True, True, True, id="No force, diff, local file change -> expect prompt, validation, expect update."
+        ),
+        pytest.param(
+            False, True, True, True, False, False, id="No force, diff, local file change -> expect prompt, no validation, expect update."
```

Comment:
```suggestion
            False, True, True, True, False, False, id="No force, diff, local file change -> expect prompt, no validation, no update."
```
I think the description here didn't quite match the expectation